### DISCUSSION TITLE
How can i match the class generated with the style?

### DISCUSSION THREAD

**Discussion by olivernosto** [glromeo/esbuild-sass-plugin#97] [Q&A] [answered]
Hi everyone!
When trying to use postcss modules,  for some reason the class generated doesnt match the style.

The example is in vugc-logo-block where the prefixes are not generating from the style to the actual class.

<img width="913" alt="image" src="https://user-images.githubusercontent.com/107518060/195213000-f9e8b212-d3e5-4727-9810-21840d0deff5.png">

This is how I am using the SASS Plugin at the moment:

      sassPlugin({
        type: 'css',
        transform: postcssModules({
            generateScopedName: '[hash:base64:8]--[local]',
            localsConvention: 'camelCaseOnly',
          })
      }),
      
Any suggestions how i can configure this correctly?

**Comment by glromeo** [comment, marked as answer]
@olivernosto  I have tried your scope name with the css-modules example in the test/fixtures and all seems OK to me...
how are you actually using the module?

**Reply by olivernosto** [reply]
Apologies @glromeo - Turns out it was using a static css name in a styled component. Working with some legacy stuff so trying to wrap my head around moving the build system across.

Well done with the tool, i will close this but as a reference for other users:

**Imports should be referenced via CSS file if a module, static css class names will not transform**
